xep-0402.xml

```diff
@@ -25,6 +25,19 @@
         <discuss>standards</discuss>
         &dcridland;
         &jcbrand;
+  <revision>
+    <version>0.3.0</version>
+    <date>2019-09-28</date>
+    <initials>egp</initials>
+    <remark><ul>
+      <li>Fix examples.</li>
+      <li>Explain the expected workflow better.</li>
+      <li>Encourage clients to set pubsub#send_last_published_item to never and pubsub#max_items to some high value.</li>
+      <li>Add examples about notifications and +notify.</li>
+      <li>Request clients to add notify='1' on retract requests, so they do trigger notifications (see <link url='https://xmpp.org/extensions/xep-0060.html#publisher-delete-success-notify'>XEP-0060 §7.2.2.1</link> for a rationale).</li>
```

Comment:
Nitpick: `retract requests` should probably be `retraction requests` to make it grammatically correct, alternatively `retract` should be put in quotes.